packages/valory/skills/abstract_round_abci/tests/test_base.py

```diff
@@ -203,13 +204,13 @@ def __init__(self, other: Any):
 def test_base_tx_payload() -> None:
     """Test BaseTxPayload."""
 
-    payload = BasePayload(sender="sender")
+    payload = PayloadA(sender="sender")
```

Comment:
This worked while it should not have. This is because `ABC` is present in the `bases`
https://github.com/valory-xyz/open-autonomy/blob/29050ce6be6165db6b6b93a496ec8ffffa0ef23c/packages/valory/skills/abstract_round_abci/tests/test_base.py#L143-L144
Hence we return early here
https://github.com/valory-xyz/open-autonomy/blob/29050ce6be6165db6b6b93a496ec8ffffa0ef23c/packages/valory/skills/abstract_round_abci/base.py#L158-L164